machine.go

```diff
@@ -405,45 +449,70 @@ func (m *Machine) writerKernelModules(w *writerhelper.WriterHelper, moddir strin
 			"modules.symbols",
 			"modules.symbols.bin",
 			"modules.builtin.bin",
-			"modules.devname")
+			"modules.devname"}
 
-	// build a list of built-in modules so that we don’t attempt to copy them
-	var builtinModules = make(map[string]bool)
 
-	f, err := os.Open(path.Join(moddir, "modules.builtin"))
-	if err != nil {
-		return err
+	for _, v := range modfiles {
+		if err := w.CopyFile(moddir + "/" + v); err != nil {
+			return err
```

Comment:
Aside: copying all the "modules.*" files is a little optimistic and off. Since they will include details for modules which are not bundled/copied over. I think the right thing is to use `depmod`.

Regardless, this is a minor, pre-existing, issue which should be kept in separate MR.